app/Rules/EmailBlacklist.php

```diff
@@ -46,11 +59,8 @@ protected function shouldUpdate(): void
     {
         $autoupdate = config('email-blacklist.auto-update');
 
-        try {
-            if ($autoupdate && !cache()->has(config('email-blacklist.cache-key'))) {
-                EmailBlacklistUpdater::update();
-            }
-        } catch (InvalidArgumentException) {
+        if ($autoupdate && !cache()->has(config('email-blacklist.cache-key'))) {
+            EmailBlacklistUpdater::update();
```

Comment:
We haven't hit the CI error caused by the email blacklist updater for awhile, but if we start seeing it again, we'll know this try-catch was what's been fixing it.